src/main/java/com/shurjomukhi/Shurjopay.java

```diff
@@ -175,7 +133,7 @@ public VerifiedPayment verifyPayment(String spTxnId) throws ShurjopayException {
 			verifiedPaymentReq.put("order_id", spTxnId);
 
 			String requestBody = prepareReqBody(verifiedPaymentReq);
-			HttpRequest request = postRequest(requestBody, Endpoints.VERIFIED_PMNT.title(), true);
+			HttpRequest request = postRequest(requestBody, "verification", true);

```

Comment:
API URL endpoint is not understandable from this code. Easy to make mistake in future.
Also, any URL path should start with '/'.

Reply:
Fixed it with constructor level.